Classes/Fluid/Unit/ViewHelpers/ViewHelperBaseTestcase.php

```diff
@@ -98,7 +98,7 @@ protected function setUp()
         $this->controllerContext->expects($this->any())->method('getUriBuilder')->will($this->returnValue($this->uriBuilder));
         $this->controllerContext->expects($this->any())->method('getRequest')->will($this->returnValue($this->request->reveal()));
         $this->arguments = [];
-        $this->renderingContext = $this->getAccessibleMock(\TYPO3\CMS\Fluid\Tests\Unit\Core\Rendering\RenderingContextFixture::class, ['getControllerContext']);
+        $this->renderingContext = $this->getAccessibleMock(\TYPO3Fluid\Fluid\Tests\Unit\Core\Rendering\RenderingContextFixture::class, ['getControllerContext', 'setControllerContext']);
```

Comment:
This isn't really fixing the class name instead it is changing the whole class to another class with different functionality. If you want to use the TYPO3Fluid rendering context I guess you could just use vendor/typo3fluid/fluid/tests/Unit/ViewHelpers/ViewHelperBaseTestcase.php instead of this one, can't you?